packages/sdk/src/blockFinder.ts

```diff
@@ -31,7 +32,14 @@ export default class BlockFinder<T extends { number: number; timestamp: number |
       const initialBlock = this.blocks[0] as WithoutStringTimestamp<T>;
       const cushion = 1.1;
       // Ensure the increment block distance is _at least_ a single block to prevent an infinite loop.
-      const incrementDistance = Math.max(await estimateBlocksElapsed(initialBlock.timestamp - timestamp, cushion), 1);
+      // We double this distance to reduce the number of iterations in the following loop and increase the chance
+      // that the first block we find sets a floor for the target timestamp. The loop converges on the correct block
+      // slower than the following incremental search performed by `findBlock`, so we want to minimize the number of
+      // loop iterations in favor of searching more blocks over the `findBlock` search.
+      const incrementDistance = Math.max(
+        (await estimateBlocksElapsed(initialBlock.timestamp - timestamp, cushion, this.chainId)) * 2,
```

Comment:
I just realized this is what the cushion does. Now that I look at it, we're already doubling (cushion is 1.1, which means we multiply by 2.1).

So we might not need this multiplication logic. Thoughts?

Reply:
ah yeah you're right, should we increase cushion to 2?